src/components/PrivacyRecommendations/Instructions.vue

```diff
@@ -18,8 +18,9 @@ const title = computed(() => {
     <ol class="ml-4 list-decimal">
       <li>
         <p class="flex items-center">
-          Click the&nbsp;<em class="inline-flex"><IcRoundMenu /></em>&nbsp;icon in the top right
-          corner of the browser
+          Click the
```

Comment:
You need a non-breaking space here too 
<img width="120" alt="image" src="https://user-images.githubusercontent.com/1008390/224696442-7de57542-380f-4055-b0b1-9900638856e1.png">

Reply:
Fixed.